app/src/main/java/com/example/iphonesearchapi/viewmodel/detailViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.iphonesearchapi.viewmodel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.iphonesearchapi.model.Result
+import com.example.iphonesearchapi.model.ResultOf
+import com.example.iphonesearchapi.network.IphoneApiService
+import kotlinx.coroutines.launch
+
+class detailViewModel(private val serviceUtil: IphoneApiService): ViewModel() {
```

Comment:
Name should be in title case
```suggestion
class DetailViewModel(private val serviceUtil: IphoneApiService): ViewModel() {
```